pkg/controller/backup_repository_controller.go

```diff
@@ -237,29 +258,34 @@ func (r *BackupRepoReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	return ctrl.Result{}, nil
 }
 
-func (r *BackupRepoReconciler) getIdentiferByBSL(ctx context.Context, req *velerov1api.BackupRepository) (string, error) {
-	loc := &velerov1api.BackupStorageLocation{}
+func (r *BackupRepoReconciler) getBSL(ctx context.Context, req *velerov1api.BackupRepository) (velerov1api.BackupStorageLocation, error) {
+	loc := new(velerov1api.BackupStorageLocation)
 
 	if err := r.Get(ctx, client.ObjectKey{
 		Namespace: req.Namespace,
 		Name:      req.Spec.BackupStorageLocation,
 	}, loc); err != nil {
-		return "", errors.Wrapf(err, "error to get BSL %s", req.Spec.BackupStorageLocation)
+		r.logger.Errorf("Fail to get BSL for BackupRepository %s: %s", req.Namespace+"/"+req.Name, err.Error())
```

Comment:
This log doesn't look necessary since this is like a helper function and the error has been returned to the caller

Reply:
Deleted.